### PR TITLE
French Translation

### PR DESCRIPTION
**Pull request by clouedoc**
untested, not sure about the Go implementation

### CODE REVIEW

**Comment by auyer** [comment]
Thanks for your contribution ! Don't worry about the tests and the implementation, I can do them myself.
There is only one issue that I had forgotten about. There were a couple changes in the develop branch planned for version `1.0`, including a couple new messages.

I just made the necessary changes, and inserted the new fields into your FR translation part (in english).
Can you translate the `SummonHelp` key, and one new line in the `HelpMessage`  ? 
Thanks!

File: public/messages.yaml [hunk 1]
```diff
@@ -89,3 +91,40 @@ PT:
...
+FR :
+  LangSet : "Tu as choisis le français [FR]. Tu peux toujours changer de langage en utilisant la même commande."
+  CantFindChannel: "Désolé, je ne peux pas trouver le canal %s."
+  CantFindUser: "%s, tu dois être connecté(e) à un canal pour que je puisse te trouver. Utilise \' %s move\' pour obtenir de l'aide."
+  GeneralHelp: "M'as-tu appelé, %s ? Utilise \' %s help\' pour obtenir de l'aide."
+  NotInGuild: "Hey %s, tu devrais utiliser cette commande dans une guilde uniquement, pas dans un chat privé."
+  HelpMessage: |
+      Utilise cette commande pour déplacer des utilisateurs d'un canal vocal à un autre !
+      Tape \' %s move\' pour plus de détails.
+      \' %s summon\' summon all users connected to voice channels into one.
```
Comment (auyer) [inline public/messages.yaml]: Can you translate this new line please? L 103

File: public/messages.yaml [hunk 2]
```diff
@@ -89,3 +91,40 @@ PT:
...
+    The summon feature will move all users from all voice channels to the specified channel.
+
+    %s summon - Display this help message and the list of available channels.
+    %s summon all - Move all users from all channels to your current channel.
+    %s summon all afk - Move all users from all channels to your current channel. Even from the AFK channel.
+    %s summon *destination* - Move all users from all channels to the *destination* channel.
+    %s summon *destination* afk - Move all users from all channels to the *destination* channel. Even from the AFK channel.
+
+    List of available channels \:
+    %s
```
Comment (auyer) [inline public/messages.yaml:173]: Can you translate this new key please? L117 - L127

**Review by auyer — CHANGES_REQUESTED** [review]

File: public/messages.yaml [hunk 3]
```diff
@@ -8,6 +8,10 @@ LANG:
     [PT/BR] Olá ! Obrigado por adicionar este bot.
         Configure a linguagem em sua guild digitando \' %s lang\' e escolhendo uma das opçoes.
         Visite o site e adicione mais níveis de poder! http://massmover.github.io
+
+    [FR] Salut ! Merci d'avoir ajouté ce bot.
+        Choisis le language de ta guilde en utilisant la commande \' %s lang\' et en choisissant une des options.
+        Visite le site afin de déplacer, toujours plus ! http://massmover.github.io
   LangSetupMessage: |
     1 - [EN] Type \' %s lang\' followed by \'EN\' or \'1\' to select the English language
```
Comment (auyer) [inline public/messages.yaml]: This line needs translation as well.

Comment (auyer) [inline public/messages.yaml:20]: Almost forgot about this line (L 12).  It needs translation as well.

**Review by auyer — CHANGES_REQUESTED** [review]

**Comment by clouedoc** [comment]
@auyer I think there's nothing left to translate !

**Review by auyer — APPROVED** [review]

**Comment by auyer** [comment]
Thank you so much ! Merci beaucoup!

**Pull request merged**